File: Aula 05/adivinhe.py
from random import *

num =  randint(0,10)
i = 0
controle = 0

while (controle == 0 ):
  i = i + 1
  x = int(input("Entre com um número inteiro "))
  if num == x:
    print("Parabéns, você acertou em ", i , " tentativas" )
    controle = 1
  elif num > x:
    print("O número pensado é maior")
  else:
    print("O numero pensado é menor")
    
    

File: Aula 07/adivinhe.java
import java.util.Scanner;
import java.util.Random;
class Main {
  public static void main(String[] args) {
        Random gerador = new Random();
        int num = gerador.nextInt(20);
        int x, i = 0;
        Scanner sc = new Scanner(System.in);
        do{
          i++;
          System.out.println("Digite um número entre 0 e 20:");
          x = sc.nextInt();
          if(x == num){
            System.out.println("Parabéns, você acertou em " + i + " tentativas" );
          } else if (x < num) {
            System.out.println("O número pensado é maior");
          } else {
            System.out.println("O número pensado é menor");
          }
        } while (x != num); 
        sc.close();
  }
}

